### PR TITLE
Added very basic linked app docs

### PR DESCRIPTION
**Pull request by orangejenny**
If there are already technical linked app docs, please let me know - I grepped a bit for READMEs and files in docs directories, but I didn't search exhaustively.

### CODE REVIEW

**Comment by orangejenny** [comment]
killed build

**Review by gbova — APPROVED** [review]

**Review by calellowitz — APPROVED** [review]

File: corehq/apps/linked_domain/README.md [hunk 1]
```diff
@@ -30,3 +30,24 @@ DomainLink.link_domains('linked_domain_name', 'master_domain_name', remote_detai
 On remote HQ, enable `linked_domains` feature flag and navigate to `project settings > Linked Projects` page which has a UI to pull changes from master domain for custom data fields for Location, User and Product models, user roles and feature flags/previews.
 
 Linked apps can be setup between linked domains by running `link_app_to_remote` command on linked domain.
+
+# Linked Applications
+
+Linked applications predate linked domains and can be used independently of linked domains.
```
Comment (snopoke) [inline corehq/apps/linked_domain/README.md]: Not true. If you link and app the linked domain record is automatically created.

Comment (orangejenny) [reply corehq/apps/linked_domain/README.md]: thanks, fixed

**Comment by orangejenny** [comment]
Added note on fields that are independent on the linked app (build profiles, etc)

**Review by emord — APPROVED** [review]

**Pull request merged**